test/e2e/page-objects/pages/snap-simple-keyring-page.ts

```diff
@@ -168,9 +168,7 @@ class SnapSimpleKeyringPage {
     console.log(
       'Approve/Reject snap account transaction on Snap Simple Keyring page',
     );
-    await this.driver.clickElementAndWaitToDisappear(
-      this.confirmationSubmitButton,
-    );
+    await this.driver.clickElement(this.confirmationSubmitButton);
```

Comment:
should this be like this, to avoid race conditions?

```suggestion
    await this.driver.clickElementAndWaitForWindowToClose(
      this.confirmationSubmitButton,
    );
```